src/BaGet.UI/package.json

```diff
@@ -13,11 +13,11 @@
   "dependencies": {
     "bootstrap": "3.3.7",
```

Comment:
Could you bump `bootstrap` to 3.4.1 to address this: https://github.com/advisories/GHSA-9v3m-8fp8-mj99

This should do the trick: ` npm install bootstrap@3.4.1`

Reply:
Ok.